rts/at/util/src/main/java/org/jboss/jbossts/star/util/logging/at18NLogger.java

```diff
@@ -0,0 +1,124 @@
+package org.jboss.jbossts.star.util.logging;
+
+import static org.jboss.logging.Logger.Level.ERROR;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import javax.transaction.xa.XAException;
+
+//import javax.json.JsonObject;
+
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * at18N log messages for the rest-at module.
+ */
+@MessageLogger(projectCode = "AT")
+public interface at18NLogger {
+	
+
+    /*
+        Message IDs are unique and non-recyclable.
+        Don't change the purpose of existing messages.
+          (tweak the message text or params for clarification if you like).
+        Allocate new messages by following instructions at the bottom of the file.
+     */
+	@LogMessage(level = ERROR)
+    @Message(id = 27001, value = "REST-AT transaction is required for this request. '%s'")
+    void error_rest_at_transactionrequired(int statusCode, @Cause Throwable t);
+
+    @LogMessage(level = ERROR)
+    @Message(id = 27002, value = "REST-AT transaction is not supported by this resource. '%s'")
+    void error_rest_at_transactionnotsupported(int statusCode,@Cause Throwable t);
```

Comment:
it would be better to have `space` after the comma (`,`). The REST AT module does work with a checkstyle but this is usual. Not necessary to change, but nice to have.